.lando.yml

```diff
@@ -20,17 +17,25 @@ tooling:
       - "--env=lando"
 
 services:
+  appserver:
+    overrides:
+      environment:
+        HASH_SALT: notsosecurehash
+  chrome:
```

Comment:
Chrome tends to be quite resource-hungry, and that has caused issues in a few cases. It would be best to keep it commented out by default.

Reply:
Done